internal/app/api/v1/testtriggers.go

```diff
@@ -0,0 +1,195 @@
+package v1
+
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/kubeshop/testkube/pkg/keymap/triggers"
+	triggerskeymapmapper "github.com/kubeshop/testkube/pkg/mapper/keymap/triggers"
+	testtriggersmapper "github.com/kubeshop/testkube/pkg/mapper/testtriggers"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
+	"github.com/kubeshop/testkube/pkg/crd"
+)
+
+// CreateTestTriggerHandler is a handler for creating test trigger objects
+func (s *TestkubeAPI) CreateTestTriggerHandler() fiber.Handler {
```

Comment:
Just idea: good place to abstract (or use existing) into some generics based HTTP REST client - too much repeated code - we could discuss @dejanzele @vsukhin about this?

Reply:
Yes, we need. it's kinda technical debt